packages/zen-bull-netting/src/ZenBullNetting.sol

```diff
@@ -707,6 +727,8 @@ contract ZenBullNetting is Ownable, EIP712, FlashSwap {
         }
         depositsIndex = k;
         isAuctionLive = false;
+
+        emit DepositAuction(_params.depositsToProcess, _params.crabAmount, _params.clearingPrice, k);
```

Comment:
How are we doing it in Crab netting, can't see the total emitted anywhere? 

Isn't that the sum of `quantity` in `TransferOsqthToMarketMakers` event, I guess it can be done through the subgraph query... not sure.